test/SlowTests/Issues/RavenDB-18426.cs

```diff
@@ -0,0 +1,78 @@
+using FastTests;
+using System.Linq;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Queries;
+using Raven.Server.Documents.Indexes.Static.Extensions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_18426 : RavenTestBase
+{
+    public RavenDB_18426(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    [Fact]
+    public void ProjectionIdTest()
+    {
+        using (var store = GetDocumentStore())
+        {
+            new DocIndex().Execute(store);
+            var requestExecutor = store.GetRequestExecutor();
```

Comment:
Also, requestExecutor is not needed here (copied from another test regarding caching).